### PR TITLE
[twilio-video] Add Network Quality stats for Participant

### PR DESCRIPTION
**Pull request by Nifled**
Based on the [`NetworkQualityStats](https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/NetworkQualityStats.html) API. Useful for analytics.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/NetworkQualityStats.html

### CODE REVIEW

File: types/twilio-video/index.d.ts [hunk 1]
```diff
@@ -170,6 +171,38 @@ export class MediaServerRemoteDescFailedError extends TwilioError {
     code: 53403;
     message: 'Server is unable to apply a remote media description';
 }
+export class NetworkQualityAudioStats extends NetworkQualityMediaStats { }
+export class NetworkQualityBandwidthStats {
+    actual: number | null;
+    available: number | null;
+    level: NetworkQualityLevel;
```
Comment (ktsn) [inline types/twilio-video/index.d.ts]:
```suggestion
    level: NetworkQualityLevel | null;
```
https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/NetworkQualityBandwidthStats.html

File: types/twilio-video/index.d.ts [hunk 2]
```diff
@@ -170,6 +171,38 @@ export class MediaServerRemoteDescFailedError extends TwilioError {
...
+}
+export class NetworkQualityFractionLostStats {
+    fractionLost: number | null;
+    level: NetworkQualityLevel;
+}
+export class NetworkQualityLatencyStats {
+    jitter: number | null;
+    rtt: number | null;
+    level: NetworkQualityLevel;
```
Comment (ktsn) [inline types/twilio-video/index.d.ts]:
```suggestion
    level: NetworkQualityLevel | null;
```
https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/NetworkQualityLatencyStats.html

File: types/twilio-video/index.d.ts [hunk 3]
```diff
@@ -170,6 +171,38 @@ export class MediaServerRemoteDescFailedError extends TwilioError {
...
 }
+export class NetworkQualityAudioStats extends NetworkQualityMediaStats { }
+export class NetworkQualityBandwidthStats {
+    actual: number | null;
+    available: number | null;
+    level: NetworkQualityLevel;
+}
+export class NetworkQualityFractionLostStats {
+    fractionLost: number | null;
+    level: NetworkQualityLevel;
```
Comment (ktsn) [inline types/twilio-video/index.d.ts]:
```suggestion
    level: NetworkQualityLevel | null;
```

https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/NetworkQualityFractionLostStats.html

**Review by ktsn — CHANGES_REQUESTED** [review]

File: types/twilio-video/index.d.ts [hunk 4]
```diff
@@ -170,6 +171,38 @@ export class MediaServerRemoteDescFailedError extends TwilioError {
...
+export class NetworkQualityMediaStats {
+    send: NetworkQualityLevel;
+    recv: NetworkQualityLevel;
+    sendStats: NetworkQualitySendOrRecvStats | null;
+    recvStats: NetworkQualitySendOrRecvStats | null;
+}
+export class NetworkQualitySendOrRecvStats {
+    bandwidth: NetworkQualityBandwidthStats;
+    latency: NetworkQualityLatencyStats;
+    fractionLost: NetworkQualityFractionLostStats;
```
[the remark this answers is not in the document]
Comment (Nifled) [reply types/twilio-video/index.d.ts]: You're right, I totally misread the different [verbosity level specifications](https://www.twilio.com/docs/video/using-network-quality-api#enabling-in-javascript). Changing that.

[the remark this answers is not in the document]
Comment (Nifled) [reply types/twilio-video/index.d.ts]: Fixed.

**Review by ktsn — APPROVED** [review]
LGTM! 👍

**Pull request merged**